src/main/java/Task.java

```diff
@@ -0,0 +1,31 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public Task(String description, boolean b) {
+        this.description = description;
+        this.isDone = b;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public String getDescription() {
+        return this.description;
+    }
+
+    public Task changeStatus(Task t) {
```

Comment:
Since this method is a setter, maybe setStatusTrue() would be better? It would explain to readers what status you are changing to as well!